web-client/src/presenter/actions/getWorkItemsByUserAction.js

```diff
@@ -1,13 +1,13 @@
-import { state } from 'cerebral';
-import _ from 'lodash';
-
-export default async ({ applicationContext, get }) => {
+/**
```

Comment:
Just noting that a lot of these functions don't have the docstrings at the top like this one does. You may want to make sure they're there.

Reply:
we can try to go through and make sure we have jsdocs on our methods.

Reply:
Instead of adding all of these docstrings now, since there are a considerable number, let's agree that the work will be done over the next sprint or two.

Reply:
👍 That sounds good to me.